### ISSUE TITLE
A link with several lines and Nvda treatment.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5053]
**Reported by khsruru on 2015-04-24 07:19**
Hello, I’m Hyongsop Kim in Korea.
I have one suggestion about a tag in Internet Explorer of Nvda.
If a tag is consist of several lines, Nvda treats each line as each link in case using arrow keys.
For example, let’s suppose that below source is exist.
<a class="NPI=a:list" href="/best/list.nhn?novelId=195441">
                                <div class="pic"><img width="150" height="225" alt="test" src="http://novel.phinf.naver.net/20140509_62/novel_139960955692444wc1_JPEG/13995114058045B15D.jpg?type=f154_231_2"></div>
                                <div class="skin"></div>

```
                            <div class="info">
                                <p class="genre">romantic</p>
                                <p class="subj">Test link</p></a>
```

In this example, in fact there is only one link, but it is divided some lines.
But Nvda reads as if there are some different links.
In that case, screen reader users think that there are different links unless they press tab key.
I also thought that how can we solve this problem.
My idea is that inform only one time as a link.
In other words, if one link has three lines, when focus is reached first line, Nvda says link.
But from next line, Nvda doesn’t say ‘link’.
Then users don’t be confused as several links.

Please check this issue.
Thank you.

**Comment by ehollig** [comment]
Possible duplicate of #4947 and #374

**Comment by leonardder** [comment]
Thanks for this, I stumbled upon this issue and knew that there was a duplicate, but didn't know its number. Closing as duplicate of #374
